scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -2,38 +2,77 @@
 -- Area: Southern San d'Oria
 --  NPC: Kipopo
 -- Type: Leathercraft Synthesis Image Support
--- !pos -191.050 -2.15 12.285 230
+-- !pos  -191.050 -2.15 12.285 230
 -----------------------------------
-require("scripts/globals/status");
-require("scripts/globals/crafting");
-local ID = require("scripts/zones/Southern_San_dOria/IDs");
+require("scripts/globals/status")
+require("scripts/globals/crafting")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+local ID = require("scripts/zones/Southern_San_dOria/IDs")
 -----------------------------------
 
 function onTrade(player,npc,trade)
+    local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
+
+    if player:hasKeyItem(dsp.ki.TORN_PATCHES_OF_LEATHER) and sayItWithAHandbagCS == 2 then
+        if npcUtil.tradeHasExactly(trade, {2012, 850, 816}) then
```

Comment:
No need to have a separate if here can make it all function

Reply:
All 1 line i mean

Reply:
This is for demonstration purposes:
```lua
    if 
        npcUtil.tradeHasExactly(trade, {2012, 850, 816})
        and player:hasKeyItem(dsp.ki.TORN_PATCHES_OF_LEATHER) 
        and sayItWithAHandbagCS == 2 
    then
```
to avoid double `if` and a wide line.

This one really isn't that wide I guess, but wanted to show you that if need be you can break them up this way and it still works.

Reply:
Thank you for noticing this. I changed it to two lines as it was going over 120 characters. 😸